components/generic/RelatedCollections.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <b-container
+    v-if="relatedCollections.length > 0"
+    data-qa="related collections"
+  >
+    <h2 class="related-heading text-uppercase mt-4 mb-2">
+      {{ $t('relatedCollections') }}
+    </h2>
+    <RelatedChip
+      v-for="relatedCollection in relatedCollections"
+      :id="relatedCollection.id"
+      :key="relatedCollection.id"
+      :link-gen="suggestionLinkGen"
+      :title="relatedCollection.prefLabel[$i18n.locale]"
+    />
+  </b-container>
+</template>
+
+<script>
+  import RelatedChip from './RelatedChip';
+  import { getEntitySuggestions, getEntityTypeHumanReadable, getEntitySlug } from '../../plugins/europeana/entity';
+  import { mapGetters } from 'vuex';
+
+  export default {
+    name: 'RelatedCollections',
+
+    components: {
+      RelatedChip
+    },
+
+    props: {
+      query: {
+        type: String,
+        default: ''
+      }
+    },
+
+    data() {
+      return {
+        relatedCollections: []
+      };
+    },
+
+    computed: {
+      ...mapGetters({
+        apiConfig: 'apis/config'
+      })
+    },
+
+    watch: {
+      query() {
+        this.$fetch();
+      }
+    },
+
+    fetch() {
+      this.getSearchSuggestions(this.query);
+    },
+
+    mounted() {
+      this.$fetch();
+    },
+
+    methods: {
+      async getSearchSuggestions(query) {
+        const suggestions = await getEntitySuggestions(query, {
+          language: this.$i18n.locale
+        });
+        this.relatedCollections = suggestions.slice(0, 4);
```

Comment:
This results in 10 suggestions coming back from the API. Let's limit the number requested, with:
```diff
diff --git a/components/generic/RelatedCollections.vue b/components/generic/RelatedCollections.vue
index 15a142fd..95d0f696 100644
--- a/components/generic/RelatedCollections.vue
+++ b/components/generic/RelatedCollections.vue
@@ -63,10 +63,10 @@
 
     methods: {
       async getSearchSuggestions(query) {
-        const suggestions = await getEntitySuggestions(query, {
-          language: this.$i18n.locale
+        this.relatedCollections = query === '' ? [] : await getEntitySuggestions(query, {
+          language: this.$i18n.locale,
+          rows: 4
         });
-        this.relatedCollections = suggestions.slice(0, 4);
       },
 
       suggestionLinkGen(id, prefLabel) {
diff --git a/plugins/europeana/entity.js b/plugins/europeana/entity.js
index cb9ddffb..c9cf311f 100644
--- a/plugins/europeana/entity.js
+++ b/plugins/europeana/entity.js
@@ -33,14 +33,13 @@ function entityApiUrl(endpoint) {
  * Get entity suggestions from the API
  * @param {string} text the query text to supply suggestions for
  * @param {Object} params additional parameters sent to the API
- * @param {string} params.language language(s), comma-separated, to request
  */
 export function getEntitySuggestions(text, params = {}) {
   return axios.get(entityApiUrl('/suggest'), {
     params: {
+      ...params,
       text,
       type: 'agent,concept',
-      language: params.language,
       scope: 'europeana',
       wskey: config.entity.key
     }
```